tests/usecase/test_db_config_options.py

```diff
@@ -0,0 +1,85 @@
+import os
+from database_config import get_database_config
+
+
+class TestDatabaseConfigOptions:
+    def clean_env(self):
+        """
+        Setting different environment variables for test in the same process
+        can lead to flaky tests.
+        """
+        if "DATABASE_URL" in os.environ.keys():
+            del os.environ["DATABASE_URL"]
+
+        for key in filter(
+            lambda x: x.startswith("MXRUNTIME_Database"),
+            list(os.environ.keys()),
```

Comment:
No need for the `list(...)` conversion, and the `.keys()` call, `os.environ` is already iterable by the keys.
Alternatively to filter you could also use a generator expression: `(k for k in os.environ if k.startswith("..."))`

Reply:
without the list it throws "RuntimeError: dictionary changed size during iteration"

Reply:
I overlooked that, but makes sense because you are deleting items.